datafusion/core/src/datasource/file_format/json.rs

```diff
@@ -231,17 +231,21 @@ mod tests {
         projection: Option<Vec<usize>>,
         limit: Option<usize>,
     ) -> Result<Arc<dyn ExecutionPlan>> {
+        let store_root = env!("CARGO_MANIFEST_DIR");
         let filename = "tests/jsons/2.json";
         let format = JsonFormat::default();
-        scan_format(&format, ".", filename, projection, limit).await
+        scan_format(&format, store_root, filename, projection, limit).await
     }
 
     #[tokio::test]
     async fn infer_schema_with_limit() {
         let store = Arc::new(LocalFileSystem::new()) as _;
-        let filename = "tests/jsons/schema_infer_limit.json";
         let format = JsonFormat::default().with_schema_infer_max_rec(Some(3));
 
+        let store_root = std::path::Path::new(env!("CARGO_MANIFEST_DIR"));
+        let filename = store_root.join("tests/jsons/schema_infer_limit.json");
+        let filename = filename.to_str().expect("Unable to get path!");
```

Comment:
Reverting these should make things work

Reply:
Ah, thank you! I missed these when reverting things. I'm pushing a new commit to this PR that I think will be releasable.

Sorry I missed these, I appreciate your help :raised_hands:

Reply:
No worries at all, for reference the reason this breaks is that paths on Windows not only don't start with `/` but when canonicalized get converted to [extended length path syntax](https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file). Its a complete nonsensical mess, in the end I punted on all this and LocalFileSystem actually just implements the file URI standard which basically behaves like paths on Linux (or paths on anything that isn't Windows :laughing: )